packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -6780,6 +6780,14 @@ export interface ISlider {
     value: number | undefined;
 }
 
+// @public (undocumented)
+export interface ISliderEnableMarksFormat {

```

Comment:
Suggest making the name more clear to communicate that this will be the objects that can be passed in an array for custom marks. "ISliderMarksArrayFormat" or something of that sort.